src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -243,6 +243,21 @@ <h5 jhiTranslate="artemisApp.exercise.export.options"></h5>
                             </ng-template>
                         </ngx-datatable-column>
                     }
+                    <ngx-datatable-column prop="" [sortable]="true" [minWidth]="100" [width]="100">
+                        <ng-template ngx-datatable-header-template>
+                            <span class="datatable-header-cell-wrapper d-inline-block w-100 text-center" (click)="controls.onSort('results?.last()?.assessmentNote?.note')">
+                                <fa-icon [icon]="faComment" [ngbTooltip]="'artemisApp.assessment.assessmentNote' | artemisTranslate" />
+                                <fa-icon [icon]="controls.iconForSortPropField('results?.last()?.assessmentNote?.note')" />
+                            </span>
+                        </ng-template>
+                        <ng-template ngx-datatable-cell-template let-value="value">
+                            @if (value.results?.last()?.assessmentNote?.note?.trim()) {
+                                <span class="w-100 text-center">
+                                    <fa-icon [ngbPopover]="(value.results?.last()?.assessmentNote)!.note" [triggers]="'hover'" [icon]="faComment" [container]="'body'" />
```

Comment:
```suggestion
                                    <fa-icon [ngbPopover]="value.results?.last()?.assessmentNote?.note" [triggers]="'hover'" [icon]="faComment" [container]="'body'" />
```

Reply:
@rabeatwork mentioned in the testing session today, that there is also no need for the `!` here.

Reply:
I removed now the `!` with 3e9aa6b8937f904ba98706b2b30cccb626e10bbe.